src/Components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,24 @@
+import Button from '@mui/material/Button'
+
+type ActionButtonProps = {
+    label: string
+    onClickEvent: () => void
```

Comment:
You can reference to `ButtonProps['onClick']` to get better typings for the `event` object like this:

```
import Button, { ButtonProps } from '@mui/material/Button'

type ActionButtonProps = {
    label: string
    onClick: ButtonProps['onClick']
}
```